recipes/dgl/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "dgl" %}
+{% set version = "1.1.0" %}
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmlc/dgl
+  git_rev: {{ version }}
+  patches:
+    - conda-build.patch
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  rpaths:
+    - lib/
+      # PyTorch libs are in site-packages intead of with other shared objects
+    - {{ SP_DIR }}/torch/lib/
+  missing_dso_whitelist:
+    - '*/libcuda.*'  # [linux64]
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    - sysroot_{{ target_platform }} 2.17  # [linux]
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - git 
+
+
+  host:
+    - python =3.10
+    - setuptools
+    - cython <3
+    - dlpack ==0.7
+    - dmlc ==0.5
+    - gklib ==5.1.1
+    - metis ==5.1.1
+    - nanoflann
+    - parallel-hashmap
+    - pytorch  # duplicate dep to pick up conda-forge pinnings
+    - pytorch * *{{ torch_proc_type }}*
+
+  run:
+    - python =3.10
+    - numpy
+    - scipy
+    - networkx
+    - requests
+    - tqdm
+    - pytorch * *{{ torch_proc_type }}*
+    - psutil
+    - tensorflow
+
+
+test:
+  imports:
+    - dgl
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/dmlc/dgl
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
Since apparently you are statically linking libraries from the third-party folder, you need to list the license files for those libaries here in addition to the DGL license.

Reply:
Yes, thank you! I forgot I needed to do that :smiley: